src/components/Reports/ReportEdit.tsx

```diff
@@ -165,13 +196,40 @@ export default function ReportEdit(): JSX.Element {
           additionalRightButtons={rightButtons}
         >
           {report &&
+            draftReport &&
             (showAnnual ? (
               <ReportFormAnnual editable={true} report={report} />
             ) : (
-              <ReportForm editable={true} report={report} onUpdateReport={updateReport} />
+              <ReportForm
+                editable={true}
+                draftReport={draftReport}
+                onUpdateReport={updateReport}
+                allSeedbanks={getAllSeedBanks(organization)
+                  .filter((f) => !!f)
+                  .map((f) => seedbankFromFacility(f!, report))}
```

Comment:
Read `allSeedbanks` from `report`, which has the original saved data. If this isn't done, then unchecking and rechecking a seedbank will cause its data to be cleared.